README.md

```diff
@@ -985,29 +983,15 @@ Now let's add logic to generate embeddings automatically anytime new rows are ad
     npx supabase functions new embed
     ```
 
-1.  In `embed/index.ts`, create an embedding pipeline using Transformers.js.
+1.  In `embed/index.ts`, create an inference session using Supabase's AI inference engine.
 
     ```tsx
     import { createClient } from '@supabase/supabase-js';
-    import { env, pipeline } from '@xenova/transformers';
 
-    // Configuration for Deno runtime
-    env.useBrowserCache = false;
-    env.allowLocalModels = false;
-
-    const generateEmbedding = await pipeline(
-      'feature-extraction',
-      'Supabase/gte-small'
-    );
+    const model = new Supabase.ai.Session('gte-small');
```

Comment:
let's add this to the top of the examples, so people copying the example will get the type definitions.

```
// Setup type definitions for built-in Supabase Runtime APIs
/// <reference types="https://esm.sh/@supabase/functions-js/src/edge-runtime.d.ts" />
```

Reply:
Good call, done